<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGraphicsView" name="GraphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="MenuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="MenuGra">
    <property name="title">
     <string>Gra</string>
    </property>
    <addaction name="ActionNowa"/>
    <addaction name="ActionPokazRozwiazanie"/>
    <addaction name="ActionZakoncz"/>
   </widget>
   <widget class="QMenu" name="MenuPomoc">
    <property name="title">
     <string>Pomoc</string>
    </property>
    <addaction name="ActionPomoc"/>
    <addaction name="ActionAutor"/>
   </widget>
   <addaction name="MenuGra"/>
   <addaction name="MenuPomoc"/>
  </widget>
  <widget class="QToolBar" name="MainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="ActionNowa">
   <property name="text">
    <string>Nowa</string>
   </property>
  </action>
  <action name="ActionPokazRozwiazanie">
   <property name="text">
    <string>Pokaż rozwiązanie</string>
   </property>
  </action>
  <action name="ActionZakoncz">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
  <action name="ActionPomoc">
   <property name="text">
    <string>Pomoc</string>
   </property>
  </action>
  <action name="ActionAutor">
   <property name="text">
    <string>Autor</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
